frontend/public/components/monitoring/utils.ts

```diff
@@ -63,7 +63,7 @@ export const getAlertsAndRules = (
   });
 
   // Add `rule` object to each alert
-  const alerts = _.flatMap(rules, (rule) => rule.alerts.map((a) => ({ rule, ...a })));
+  const alerts = _.flatMap(rules, (rule) => rule.alerts?.map((a) => ({ rule, ...a })));
```

Comment:
Now that https://github.com/thanos-io/thanos/pull/2888 has merged, `rule.alerts` should always be an array, so we shouldn't need to add the `?`.

Reply:
I will update it once this change gets reflects to API.

Reply:
done